### ISSUE TITLE
"Invoke-PSUScript: Cannot retrieve the dynamic parameters for the cmdlet" after upgrade to 4.1.5

### ISSUE THREAD

**Issue by RockoTheeHut** [ironmansoftware/powershell-universal#2739]
### Version

4.1.5

### Severity

Low

### Steps to Reproduce

I have ran this on two servers and occurs on both. This did not happen prior to upgrade to 4.1.5

import-module universal -force

$Token = 'xxxxx<apptoken>xxxxxxx'

Connect-PSUServer -ComputerName https://xxxxxxxxx -AppToken $token

$Job = Invoke-PSUScript -Id 21

### Expected behavior

```console
Script executes and provides a job object
```

### Actual behavior

```console
Invoke-PSUScript: Cannot retrieve the dynamic parameters for the cmdlet. Response could not be deserialized to JSON: GET https://xxxxxxxxx/api/v1/script/21
```

### Additional Environment data

PSU 4.1.5
SQL Server 2019
Windows Server 2019

**Comment by adamdriscoll** [comment]
@RockoTheeHut - I can't reproduce this. Can you let me know what parameters are defined in script 21? Also, can you let me know if you have the latest Universal module installed?

**Comment by RockoTheeHut** [comment]
@adamdriscoll I don't have any parameters set for this.. I have a dummy job I was using to figure out how this process works and how output is returned to the calling user so we can start utilizing it. The script:
```
Write-Output "Write Output"
Write-Host "Write host"
"Print to screen"
```

As for the latest universal module, I verified that the install location for PSU is 4.1.5. I also downloaded the 4.1.5 module from the gallery and put that in the module path and loaded that to be sure and it still produces the same thing

--

Ok, so as I was typing my reply I had an idea.. and it worked. I was using a token I manually generated for my OIDC login that uses "policy" to give rights for the token. The policy is applied by a group claim in AzureAD, and in this case, it is administrator. It doesn't work.

I then used another admin token I generated for the local admin account that is hard set to administrator and the call then worked. 

Changed it back to the original, it stopped working. I then created a new token for my OIDC login but hard set it to admin and the call now works. 

It appears that setting the role to "policy" isn't working.
